src/data/upgrades.js

```diff
@@ -86,6 +86,86 @@ export const COMBAT_UPGRADES = {
 			validhunting: 5
 		}
 	},
+	combatXPBoost1: {
+		name: "Combat Learning Tier I",
+		description: "Increases the percentage of combat skill experience you get from damage you deal by 8.5%.",
+		icon: require('@/assets/art/combat/upgrades/bigbrain.png'),
+		requiredItems: {
+			money: 25000
+		},
+		upgrade: "combatXPBoost",
+		requiredUpgrades: {
+			combatXPBoost: 0
+		},
+		requiredLevels: {
+			validhunting: 10
+		}
+	},
+	combatXPBoost2: {
+		name: "Combat Learning Tier II",
+		description: "Further increases the percentage of combat skill experience you get from damage you deal by 8.5%.",
+		icon: require('@/assets/art/combat/upgrades/bigbrain.png'),
+		requiredItems: {
+			money: 50000
+		},
+		upgrade: "combatXPBoost",
+		requiredUpgrades: {
+			combatXPBoost: 1
+		},
+		requiredLevels: {
+			validhunting: 20
```

Comment:
My gut is saying a 50% increase to combat XP at only 20 validhunting tasks feels a bit too fast.

To put that into perspective, that's around ~200 kills to get, of the ~1700 you need to max validhunting.
![image](https://user-images.githubusercontent.com/16600313/89735636-12643b00-da32-11ea-91ce-31e22087a9e7.png)

That's also just 9 reset-timecan-things, if you use them to level validhunting.

I'd say for these two the 20 and 40 intervals might feel more appropriate, but I'll leave that call to @Nabski89

Reply:
I don't think this is an upgrade that should be unlocked on the first playthrough. I'm am going to move them into the chronoshop after this is merged. I will be keeping the validhunting and other item requirements because as we discussed that keeps it out of instant range for account start. I agree that it may be worth moving the validhunting requirement so it doesn't line up with what you can buy with bluespace but EH.